Pipeline/RunIDZCodeReview/codereview.properties

```diff
@@ -26,4 +26,9 @@ codereview_includedIncludeFiles=**/*.cpy
 ## The generated CR JCL uses the the LIST DD. This mapping indicates to which language a given file extension is mapped
 ## Specify the L=COBOL parameter for the Code Review List mapping
 codereview_languageMapping=COBOL :: **/*.cbl
-codereview_languageMapping=PLI :: **/*.pli
\ No newline at end of file
+codereview_languageMapping=PLI :: **/*.pli
+
+#
+## Additional datasets added to the SYSLIB concatenation for the code review application
+## codereview_syslib=JENKINS.EXTERNAL.COPY
+codereview_syslib=
```

Comment:
Can you explicitly say that it is a comma separated list of datasets.

Reply:
See update